seahorn/jobs/array_list_clear/CMakeLists.txt

```diff
@@ -2,11 +2,11 @@ add_executable(array_list_clear
   ${AWS_C_COMMON_ROOT}/source/array_list.c
   aws_array_list_clear_harness.c)
 target_compile_definitions(array_list_clear
-  PUBLIC MAX_ITEM_SIZE=2
-  MAX_INITIAL_ITEM_ALLOCATION=9223372036854775808ULL)
+  PUBLIC MAX_ITEM_SIZE=${MAX_ITEM_SIZE}
```

Comment:
I believe these definitions are not needed. They are globally set in `config.h` and in `seahorn/CMakeLists.txt`

Reply:
But the compilation will report an error: "use of undeclared identifier" when I remove it. Should each harness require to include `config.h`? Or is something wrong when those definitions passed to each cmake target?

Reply:
this is the place where these definitions should be moved into: https://github.com/seahorn/verify-c-common/blob/master/seahorn/CMakeLists.txt#L309

I can see that these particular ones are missing from there.

Long term -- yes, every harness should include `config.h`, global definition of these macros will be removed. Per instance setting of these parameters will be used to override the default. 

This is what we do for all new unit proofs. So if you are changing these ones, might as well change them to follow the new form. Or, at least, the intermediate step where all definitions are kept in one cmake file

Reply:
Never mind. Fix it!